packages/behandling-felles/src/components/ProsessStegPanel.tsx

```diff
@@ -88,6 +91,8 @@ const ProsessStegPanel: FunctionComponent<OwnProps> = ({
                     behandling,
                     alleKodeverk,
                     submitCallback: bekreftAksjonspunktCallback,
+                    featureToggles,
```

Comment:
kan du ikkje heller senda med featureToggles til den aktuelle komponenten (i staden for å senda til alle her)